entrypoint.py

```diff
@@ -141,9 +141,9 @@ def modified_search_arguments(cls, **kwargs):
 class EbooksEntryPoint(MediumEntryPoint):
     INTERNAL_NAME = "Book"
     URI = u"http://schema.org/EBook"
-EntryPoint.register(EbooksEntryPoint, "Books", default_enabled=True)
+EntryPoint.register(EbooksEntryPoint, "eBooks", default_enabled=True)
 
 class AudiobooksEntryPoint(MediumEntryPoint):
     INTERNAL_NAME = "Audio"
     URI = u"http://bib.schema.org/Audiobook"
-EntryPoint.register(AudiobooksEntryPoint, "Audio")
+EntryPoint.register(AudiobooksEntryPoint, "Audio Books")
```

Comment:
I think this is normally written as one word - "Audiobooks".

Reply:
Yeah, Ricardo has `Audio Books` in the mockup, but I think NYPL uses both conventions all over. It seems `Audiobooks` is more pervasive so I'll use that. Thanks.